lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```
@@ -730,6 +801,16 @@ states:
             targetJourney: INELIGIBLE
             targetState: INELIGIBLE
 
+  CRI_DRIVING_LICENCE_AUTH_SOURCE_CHECK_AFTER_PYI_ESCAPE:
+    response:
+      type: cri
+      criId: drivingLicence
+    parent: CRI_STATE
+    events:
+      next:
+        targetJourney: EVALUATE_SCORES
+        targetState: START
+
```

This is for same session enhanced verification mitigation. If the user backs out of the drivingLicence CRI  we'll get an access-denied event. At written this will fail the users journey. But we could instead make changes to the reset-session-identity lambda to clear out the DCMAW VC and offer the user F2F as another way to achieve a verification score.
We could potentially do this for any non CI attracting failure from driving licence.

---

Ah, so this is the specific case where the user has used a passport in the web route, but for some reason chose to use a DL for the DCMAW mitigation?

We can take it to Joe/Simon to confirm, but I think it's probably fine to let them fail at that point.

---

Yep fair - anything that isn't a `next` event here will fallback to the defailt parent states, which are all fails.